lib/index.js

```diff
@@ -33,13 +89,18 @@ function start() {
   }
   const port = findPort();
   const p = spawn(process.execPath, [require.resolve('./worker'), port, ...FLAGS], {
-    stdio: 'inherit',
+    stdio: debugMode ? ['ignore', outputFile, errorFile] : 'inherit',
     windowsHide: true,
   });
   p.unref();
   process.on('exit', () => {
     p.kill();
   });
+  client = net.connect(port, '127.0.0.1', () => {
```

Comment:
is this supposed to be connected to localhost?

Reply:
Yes, it is connecting to the local `server` process (another child process created with `spawn`), that actually handles request to c3 server.